nav2_costmap_2d/CMakeLists.txt

```diff
@@ -26,17 +26,10 @@ find_package(visualization_msgs REQUIRED)
 find_package(angles REQUIRED)
 
 remove_definitions(-DDISABLE_LIBUSB-1.0)
-find_package(Eigen3 REQUIRED)
+find_package(Eigen3 3.3 REQUIRED)
 
 nav2_package()
 
-include_directories(
-  include
```

Comment:
Readd

Reply:
The packages own include directory is now handled by the `BUILD_INTERFACE` generator for `target_include_directories. I moved that call down, because you can't call `target_include_directories` before the target exists.